megamek/src/megamek/common/Coords.java

```diff
@@ -217,6 +220,50 @@ public final boolean isXOdd() {
     public int direction(Coords d) {
         return (int) Math.round(radian(d) / HEXSIDE) % 6;
     }
+    
+    /**
+     * Returns an approximate direction in which another coordinate lies; 
+     * 0 if the coordinates are equal
+     */
+    public int approximateDirection(Coords second, int initialDirection, int previousDirection) {
```

Comment:
I wound up not using this, will likely remove it.

Reply:
This is used in the commented out code on line 1394 BoardView1 (renderApproxHexDirection)